app.py

```
@@ -102,127 +106,101 @@ def listt():
 
 
 @app.route("/api/send/<int:target_user_id>")
+# api/send/{recipient}?gift_type={電電點|抽獎券}count={count}
 def send(target_user_id):
```

Ruff

`send` is too complex (11 > 10)

(C901)